client/app/scripts/superdesk-archive/module.js

```diff
@@ -140,8 +140,8 @@ define([
          * @param {Object} desk when passed the item will be assigned to this desk instead of user's activeDesk.
          */
         this.addTaskToArticle = function (item, desk) {
-            if ((!item.task || !item.task.desk) && (desk || desks.activeDeskId && desks.userDesks)) {
-                var currentDesk = desk || _.find(desks.userDesks._items, {_id: desks.activeDeskId});
+            if ((!item.task || !item.task.desk) && (desk || desks.getCurrentDeskId() && desks.userDesks)) {
```

Comment:
In desks dropdown is not possible to select 'Personal' anymore so the content item should be added to the current selected desk on desks dropdown.
The change should be performed for the code that adds content items to 'Personal'